README.md

```diff
@@ -1,99 +1,206 @@
-
-There are more guides in [docs/](docs/), e.g. the [Guide for contributing to translations](docs/contributing-to-translations.md) and the [Docker web app guide](docs/docker-web-app-guide.md)
-
-## How to contribute with a PR
-If you would like to contribute by solving an open issue, please fork this repository and then create a branch for the fix.
-
-Once you push your code to your fork, you'll be able to open a PR into Logseq repository. For more info you can follow this guide from [GitHub docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)
-
-And here a list of some [good first issues](https://github.com/logseq/logseq/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)!
-
-## Thanks
-
-[![JetBrains](docs/assets/jetbrains.svg)](https://www.jetbrains.com/?from=logseq)
+## 🧠 Learn more
+
+- Our blog: [https://blog.logseq.com/](https://blog.logseq.com)
+- Please be sure to visit our [About page](https://blog.logseq.com/about) for the latest updates
+- Twitter: <https://twitter.com/logseq>
+- Forum: <https://discuss.logseq.com> - Where we answer questions, discuss workflows and share tips
+- Discord: <https://discord.gg/KpN4eHY>
+- 中文 Discord：<https://discord.gg/xYqcrXWymg>
+- GitHub: <https://github.com/logseq/logseq> - everyone is encouraged to report issues!
+
+<br></br>
+
+# 🫂 Contribute to Logseq
+The following is for developers and designers who want to build and run Logseq locally and
+contribute to this project.
+
+We have [a dedicated overview
+page](https://github.com/logseq/logseq/blob/master/CODEBASE_OVERVIEW.md) for Logseq's codebase
+overview and [a development practices page](docs/dev-practices.md).
+
+## 🛠️ Set up development environment
+- For setting up web app / desktop app development environment on macOS / Linux, please refer to
+[Develop Logseq](docs/develop-logseq.md).
+- For Windows users, please refer to [Develop Logseq on
+Windows](docs/develop-logseq-on-windows.md) in addition.
+
+There are more guides in [docs/](docs/), e.g. the [Guide for contributing to
+translations](docs/contributing-to-translations.md) and the [Docker web app
+guide](docs/docker-web-app-guide.md)
+
+## 🌟 How to contribute with a PR
+If you would like to contribute by solving an open issue, please fork this repository and then
+create a branch for the fix.
+
+Once you push your code to your fork, you'll be able to open a PR into Logseq repository. For
+more info you can follow this guide from [GitHub docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)
+
+And here is a list of some [good first issues](https://github.com/logseq/logseq/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)!
+
+<br></br>
```

Comment:
Non-blocking

Thanks for the amazing `README.md`! I found that there are some huge line breaks. In my opinion, the rendered heading1 (`#`) and heading2 (`##`) both have margins and a divider line, so the user can quickly figure out which section they are reading.

<img width="502" alt="image" src="https://user-images.githubusercontent.com/28241963/215058112-3b8350fd-ea79-4e9d-ba51-6ddfd57c1485.png">

Reply:
Yeah I agree, I'll change it